experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -74,13 +78,16 @@ export class TinyliciousClient {
             runtimeFactory,
             false,
         );
-        return this.getFluidContainerAndServices(container);
+        const rootDataObject = await requestFluidObject<RootDataObject>(container, "/");
+        const resources = this.getFluidResources(container, rootDataObject);
+
+        return resources;
     }
 
-    private async getFluidContainerAndServices(
+    private getFluidResources(
         container: Container,
-    ): Promise<TinyliciousResources>  {
-        const rootDataObject = await requestFluidObject<RootDataObject>(container, "/");
+        rootDataObject: RootDataObject,
```

Comment:
you can move the call to get the rootDataObject into this function.

Reply:
i moved 'rootDataObject' out of `getFluidResources` because `rootDataObject` is async and the rest are not

Reply:
oh, ok. i worked out how to do this.